### ISSUE TITLE
4.1.2 - Count user visits (Community portal)

### ISSUE THREAD

**Issue by janahoffmann** [gbhl/bhl-europe#105]
Display user visits of BHL-Europe portal on main page.

Precondition: Website analytics

**Comment by fwelter** [comment]
Count user visits for every item individually, and display on the item page how often a title was viewed before.

**Comment by janahoffmann** [comment]
@fwelter This is something we can use for the browse function...but I think this would be too much information for the result page...see issue # 60

**Comment by fwelter** [comment]
Yes I agree, a possible option for #60 but perhaps not necessary or at least not with high priority for the browse function. It is important to get information on user visits for items, for us internally to know which kind of literature is more and which disciplines/epochs etc. are less requested. Provides important background information for libraries who try to get funds for digitising literature. 
We could later also provide information in tables, such as "comparison of average user visits for literature with metadata record Language:Italian compared with average visits total", for libraries who need to show that Italian literature is highly requested, to get money for digitising Italian works.

**Comment by JiriFrank** [comment]
We can use the analytics demonstrated by Bernd during the Tervuren meeting or we have our own application.

**Comment by JiriFrank** [comment]
This is not a task for testers team.

**Comment by janahoffmann** [comment]
This is not implemented yet but could be tested: present: yes/ no

We have to reopen this issue!

**Comment by JiriFrank** [comment]
Piwik will be set up.

**Comment by JiriFrank** [comment]
According to the new rules in Germany we can use the Google analytics instead of Piwik.
